[ Gameplay Wiki ] Electricity page (#3112)
GregTechCEu/GregTechCEu-1.19: docs/content/Gameplay/Electricity.md

Certain recipes might require certain voltage levels, thus requiring a higher tier machine.

Each tier quadruples the EU value of the previous tier.

| Short Name | Long Name               | EU value      |
| ---------- | ----------------------- | ------------- |
| ULV        | Ultra Low Voltage       | 8             |
| LV         | Low Voltage             | 32            |
| MV         | Medium Voltage          | 128           |
| HV         | High Voltage            | 512           |
| EV         | Extreme Voltage         | 2,048         |
| IV         | Insane Voltage          | 8,192         |
| LuV        | Ludicrous Voltage       | 32,768        |
| ZPM        | ZPM Voltage             | 131,072       |
| UV         | Ultimate Voltage        | 524,288       |
| UHV        | Ultra High Voltage      | 2,097,152     |
| UEV        | Ultra Excessive Voltage | 8,388,608     |
| UIV        | Ultra Immense Voltage   | 33,554,432    |
| UXV        | Ultra Extreme Voltage   | 134,217,728   |

> | OPV        | Overpowered Voltage     | 536,870,912   |

Reviewer: OPV is capitalised as OpV

> | MAX        | Maximum Voltage         | 2,147,483,640 |

Reviewer: MAX is 2,147,483,648

### Amperage

Machines and energy hatches draw amps to fill their EU buffers. When running a recipe, the machine will draw from its internal buffer.

> Different machines draw different amoun of amps:

Reviewer: *amounts

| Machine                | Notes                                                |
| ---------------------- | ---------------------------------------------------- |

> | Singleblock Generators | Outputs 1A of its tier                               |

Reviewer: For consistency's sake, make it single block with a space

> | Energy Hatches         | Draws 2A (second amp mostly for extra draw for loss) |

Reviewer: Parentheses blurb unnecessary imo

| Transformers Step-Up   | Draws 4A lower voltage, outputs 1A higher voltage    |
| Transformers Step-Down | Draws 1A higher voltage, outputs 4A lower voltage    |

> | Battery Buffers Draws  | 2A per Battery, outputs 1A per Battery.              |

Reviewer: Draws needs to be in the second column

## Machine Overclocking

Machines of a higher tier than that of the recipe's voltage level can overclock recipes to speed them up.
Overclocking doubles the recipe's speed, but quadruples EU/t usage. This is also called _2/4 overclock_ or _regular overclock_.
Single block machines can only regular overclock.

A _perfect overclock_ quadruples both the recipe's speed and EU/t usage.

> Some multibocks suck as the Large Chemical Reactor do perfect overclocking by default, while some have conditional perfect overclocking.

Reviewer: *such as

Reviewer 2: This is still wrong

Multiblocks with 2 Energy Hatches of the same Voltage Tier will overclock to 1 Amps of the Voltage Tier above. (For example, an Electric Blast Furncace with 2 LV Energy Hatches (4A of LV = 4 \_ 32 = 128 EU/t) will overclock to MV (1A of MV = 1 \* 128 = 128 EU/t).)

## Wires, Cables, Cable loss

> In GregTech, energy is transferred trough wires and cables. Cables are the covered versions of wires, but anything mentioned about cables applies to wires too.

Reviewer: *through

Reviewer: Also I think this sentence could be written as "Cables are insulated wires [...]"

!!! danger

>     Non-superconductor wires will electrocute you if touched while electricity is flowing trough.

Reviewer: *through

All cables have a max Voltage, max Amperage and a Loss/Meter/Ampere.

- Cables that recieve more EU than their maximum Voltage will burn up.
- Cables that have more Amps travelling through them than their maximum Amperage will burn up.

> Each energy packet (Amp) travelling trough a cable will lose Voltage per block travelled.<br>

Reviewer: *through

For example: Tin Cable can transfer 32 EU/t with a Cable Loss of 1 EU/Amps/Meter. This means, that 1 Amp can "live" for 32 blocks before it is lost.

> Cables come in different density. A higher density can transfer a higher amount of Amps.

Reviewer: *densities

Reviewer 2: This too

> To account for cable loss, machines requiring large amounts of Amps (like the EBF), you should place these machines closer to your power use and use more dense cables. For example, an EBF that has 2 LV Energy Hatches requires 4A of LV, so using only 4A Tin Cables will barely be enough (it will be as most recipe Voltages account for some power loss) as even after travelling 1 block, each amp would already lose 1 EU.

Reviewer: Sentence structuring is funky here.
"To account for cable loss, machines requiring large amounts of Amps (like the EBF) should be placed closer to your power grid and use more dense cables"

Reviewer: "[...] as even after travelling [...]" this is also funky. Probably make it something like "Keep in mind that each amp loses 1 EU per block" in a new sentence

!!! tip

>     Wires have twice the Cable Loss as Cables.

Reviewer: Weird way to introduce it to the reader imo. "Cables have half the cable loss as wires do. Insulate your wires"

> Superconductor Wires are Wires with 0 Cable Loss. These Wires don't have a cable wariant and won't electrocute you when touching them.

Reviewer: A second Wires seems redundant, just change it to "those"

Reviewer: *variant

## Dynamo Hatches

Multiblock Generators use Dynamo Hatches to output electricity. Dynamo Hatches have 1A, 4A, 16A and 64A variants.

!!! note
    Multiblock Generators will always try to "fill" existing generated Amps before trying to generate more Amps.

## Transformers

> Transformers can either transform 1 Amps of a higher Voltage into 4 Amps of a lower Voltage, or transform 4 Amps of a lower Voltage to 1 Amps of a higher Voltage. The Mode can be swapped by right-clicking with a Screwdriver.

Reviewer: Use the terms transform up/transform down instead of into here

Transformers come in 1-4, 2-8 and 4-16 versions.

### Active Transformer

The Active Transformer is a multiblock that can transform to and from any voltage, accepting energy with an Energy Hatch and outputting energy with a Dynamo Hatch.

## Diodes

> Diode blocks can be used to limit the Amperage troughput of a network. A Diode has 5 input sides (green) and 1 output side (red). By default a Diode accepts and Amps and outputs 1 Amp. The output Amperage can be changed by right clicking with a Soft Mallet.

Reviewer: *throughput

---
title: Electricity
---

# GregTech Electricity

GregTech uses an energy system different from Forge Energy (FE) called Energy Units (EU). 1 EU converts to 4 FE, but FE cannot be converted to EU without a converter.

!!! note
    The conversion rate between EU and FE can be changed in the configs.

## Amperage and Voltage

EU comes in a system of _Voltages_ and _Amperages_.

Energy is transferred in packages. Voltage describes the size of the package, while Amperage describes how many packages are sent in parallel.

### Voltage

> GregTech tiers progression into voltage tiers. The voltage tier of a generator defines the maximum EU the generator can output. The voltage tier of a single block machine or Energy Hatch describes the maximum amount of EU that it can recieve per amp. If this value is exceeded the machine explodes.

Reviewer 2: "... The voltage tier of a singleblock machine or energy hatch describes the maximum amount of EU that it can receive per amp. If that value is exceeded, the machine explodes."

| UEV        | Ultra Excessive Voltage | 8,388,608     |
| UIV        | Ultra Immense Voltage   | 33,554,432    |
| UXV        | Ultra Extreme Voltage   | 134,217,728   |
| OpV        | Overpowered Voltage     | 536,870,912   |
| MAX        | Maximum Voltage         | 2,147,483,648 |

### Amperage

Machines and energy hatches draw amps to fill their EU buffers. When running a recipe, the machine will draw from its internal buffer.

Different machines draw different amounts of amps:

| Machine                | Notes                                                |
| ---------------------- | ---------------------------------------------------- |
| Singleblock Generators | Outputs 1A of its tier                               |
| Energy Hatches         | Draws 2A                                             |
| Transformers Step-Up   | Draws 4A lower voltage, outputs 1A higher voltage    |
| Transformers Step-Down | Draws 1A higher voltage, outputs 4A lower voltage    |
| Battery Buffers        | Draws 2A per Battery, outputs 1A per Battery.        |

## Machine Overclocking

Machines of a higher tier than that of the recipe's voltage level can overclock recipes to speed them up.

> Overclocking doubles the recipe's speed, but quadruples EU/t usage. This is also called _2/4 overclock_ or _regular overclock_.

Reviewer 2: ```suggestion
Overclocking doubles the recipe's speed, but quadruples EU/t usage. This is also called a _2/4 overclock_ or _regular overclock_.
```

> Single block machines can only regular overclock.

Reviewer 2: ```suggestion
Singleblock machines can only do regular overclocks.
```

A _perfect overclock_ quadruples both the recipe's speed and EU/t usage.

Some multibocks suck as the Large Chemical Reactor do perfect overclocking by default, while some have conditional perfect overclocking.

> Multiblocks with 2 Energy Hatches of the same Voltage Tier will overclock to 1 Amps of the Voltage Tier above. (For example, an Electric Blast Furncace with 2 LV Energy Hatches (4A of LV = 4 \* 32 = 128 EU/t) will overclock to MV (1A of MV = 1 \* 128 = 128 EU/t).)

Reviewer 2: ```suggestion
Multiblocks with 2 Energy Hatches of the same voltage tier will overclock to 1 Amp of the next voltage tier.
For example, an Electric Blast Furnace with 2 LV Energy Hatches (4A of LV = 4 \* 32 = 128 EU/t) will overclock to MV (1A of MV = 1 \* 128 = 128 EU/t).
```

## Wires, Cables, Cable loss

> In GregTech, energy is transferred trough wires and cables. Cables are the covered versions of wires, but anything mentioned about cables applies to wires too.

Reviewer 2: ```suggestion
In GregTech, energy is transferred through wires and cables. Cables are insulated wires, so most information about them applies to wires as well.
```

!!! danger
    Non-superconductor wires will electrocute you if touched while electricity is flowing through.

All cables have a max Voltage, max Amperage and a Loss/Meter/Ampere.

- Cables that recieve more EU than their maximum Voltage will burn up.
- Cables that have more Amps travelling through them than their maximum Amperage will burn up.

> Each energy packet (Amp) travelling trough a cable will lose Voltage per block travelled.<br>

Reviewer 2: ```suggestion
Each energy packet (Amp) travelling through a cable will lose the specified amount of voltage per block travelled.<br>
```

> For example: Tin Cable can transfer 32 EU/t with a Cable Loss of 1 EU/Amps/Meter. This means, that 1 Amp can "live" for 32 blocks before it is lost.

Reviewer 2: ```suggestion
For example: a 1x Tin Cable can transfer 32 EU/t with a Cable Loss of 1 EU/Meter. That means 1 Amp can "live" for 32 blocks before it's completely lost.

!!! note "Machine energy requirements"

    Most recipes require $\Large\frac{15}{16}$ of the tier's EU value to function, so an LV Tin Cable can only supply enough energy for most LV recipes from at most 2 blocks away from the energy source.
```

Author: LateX doesn't seem to render correctly in MkDocs

Author: ![image](https://github.com/user-attachments/assets/5b2afb76-b77e-4515-938d-21b025c8cc3b)

Reviewer 2: hold on I'll go check

Reviewer 2: I guess you should use `¹⁵⁄₁₆` instead.

To account for cable loss, machines requiring large amounts of Amps (like the EBF), you should place these machines closer to your power use and use more dense cables. For example, an EBF that has 2 LV Energy Hatches requires 4A of LV, so using only 4A Tin Cables will barely be enough (it will be as most recipe Voltages account for some power loss) as even after travelling 1 block, each amp would already lose 1 EU.

!!! tip
    Cables have half the cable loss as wires do. Insulate your wires!

> Superconductor Wires are Wires with 0 Cable Loss. Those don't have a cable variant and won't electrocute you when touching them.

Reviewer 2: "Superconductors are wires with 0 cable loss. They don't have a cable variant and won't electrocute you when touching them."
